### ISSUE TITLE
As a sysadmin, I want logs to be aggregated and searchable, so that I can troubleshoot

### ISSUE THREAD

**Issue by camjackson** [rabblerouser/rabblerouser-infra#3]
Currently logs are piped to a file, so accessing them means SSHing on to the box.

It would be good if they could be read and searched from a browser. E.g. logentries, papertrail, etc.

**Comment by ineffyble** [comment]
Given that we're using AWS, one option would be to have Ansible install and configure the AWS CloudWatch Logs Agent, which would them stream those logs and make them accessible via AWS Console.

**Comment by camjackson** [comment]
That sounds like a reasonable option if it means we don't have to run an ELK stack or pay for some other service. I've only ever used cloudwatch logs for builtin AWS stuff before, never for my own application logs. Have you used it for app logs before?

**Comment by camjackson** [comment]
Also my initial comment above is not correct any more, at least for application logs. It's now possible to use the docker CLI to talk directly to the docker daemon running on the server, and view the logs through there. Still not an ideal solution but at least you don't have to SSH onto the box.

**Comment by ineffyble** [comment]
Yes, at the Greens we have a k8s pod running the agent, mounting some log files (for nginx, php-fpm, Drupal) off EFS and streaming them to CloudWatch.

It's a little... simple/featureless, but it definitely works.

I had a look at Ansible roles for installing the agent, but they looked a bit undersupported - might be simple enough that we just want to write our own.
